requirements.txt

```diff
@@ -1,16 +1,2 @@
-# Requirements for automatic testing and building
```

Comment:
@CMeza99 I believe @bcl  requested to leave the requirements.txt alone for use in other areas

`Please leave requirements.txt as-is, it is useful for setting up pip in a venv, container, or travis.
I'm fine with moving to setuptools and including deps in setup.py though.`

Reply:
I was still commenting when you wrote this. I responded to the concern below. :)